src/test/java/org/prebid/server/bidder/adhese/AdheseBidderTest.java

```diff
@@ -117,8 +120,12 @@ public void makeHttpRequestsShouldModifyIncomingRequestAndSetExpectedHttpRequest
         // then
         assertThat(result.getValue())
                 .extracting(HttpRequest::getUri)
-                .containsOnly("https://ads-demo.adhese.com/json/sl_adhese_prebid_demo_-leaderboard/ag55/cigent;brussels"
-                        + "/tlall/xtdummy/xzdum-my");
+                .containsOnly("https://ads-demo.adhese.com/json");
+        assertThat(result.getValue())
+                .extracting(HttpRequest::getBody)
+                .containsExactly("{\"slots\":[{\"slotname\":\"_adhese_prebid_demo_-leaderboard\"}],"
```

Comment:
It is preferable to use `AdheseRequestBody` model instead of plain JSON.
If you're agree - please check and align all other tests.

Reply:
Sounds reasonable, I'll make the adjustments